packages/react-components/lib/robots/robot-accordion.tsx

```diff
@@ -28,6 +28,12 @@ const RobotInfo = (props: RobotInfoProps) => {
     { name: 'Name', value: robot.name },
     { name: 'Model', value: robot.model },
     { name: 'Fleet', value: fleetName },
+    {
+      name: 'Location',
+      value: `${robot.location.level_name} (${robot.location.x.toFixed(
```

Comment:
Umm seems like you added back the (x, y)?

Reply:
Oops, I thought that was on Yaw